docs/scos/dev/migration-program/migration-to-paas/paas-assessment-documents/test-coverage-is-common-e-com-flow-covered.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Test coverage: Is common E-com flow covered?
+description: This document allows you to assess if Common E-com flow is covered with tests.
+template: howto-guide-template
+---
+
+# Test coverage: Is common E-com flow covered?
+
+{% info_block infoBox %}
+
+Resources: Backend
+
+{% endinfo_block %}
+
+## Description
+
+Do a code review of the archived project received as part of the prerequisites and identify if the following
```

Comment:
What if access to the repo is provided?

Reply:
Good point. I think we can rephrase this like this: `Do a code review of the project and identify if the following functionalities were covered with tests`